docs/user-guide/sdks-using.md

```diff
@@ -0,0 +1,123 @@
+# Using Zowe SDKs
+
+Learn about using the Zowe Client Software Development Kits (SDKs). The SDKs enable you to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programamatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Node Client SDK
+- Zowe Python Client SDK
+
+## Software requirements
+
+Before you use the Python SDK, install the following software:
+
+<!-- These are listed in the python repo. I assume that the end user needs to install them prior to calling the APIs? Do they install them into their project? -->
+
+- [requests-2.22](https://github.com/psf/requests)
+- [keyring](https://github.com/jaraco/keyring)
+- [pyyaml](https://github.com/yaml/pyyaml)
+
+## Getting started
+
+To get started, download and import the SDK packages to your project. Alternatively, you can pull the packages from an online registry.
+
+### Install SDK from local package
+
+Download and install the packages.
+
+**Follow these steps:**
+
+1. Navigate to [Zowe.org Downloads](https://www.zowe.org/download.html). Select your desired programming language in the **Zowe Client SDKs** section.
+
+   The SDK is downloaded to your computer.
+
+2. Open the SDK folder, which contains packages for each function. Copy each file that you want to install and paste them into your project directory.
```

Comment:
It's likely that the packages will be inside a zip file 😋 
So, should we say unzip or do we assume users know that?

Reply:
You know what they say happens when you assume.... lol. I took your suggestion.